src/compiler/bundle/ext-format-plugin.ts

```diff
@@ -17,19 +17,19 @@ export const extFormatPlugin = (config: d.Config): Plugin => {
       // ?format= param takes precedence before file extension
       switch (format) {
         case 'url':
-          return formatUrl(config, this, code, filePath, ext);
+          return {code: formatUrl(config, this, code, filePath, ext), map: null};
```

Comment:
I added the RollupTransformHook as the return signature of this transform function in my PR  https://github.com/johnjenkins/stencil/pull/1/commits/a593372ced0ba07758a972f64f0e67d97003b6e9